logzio-sender/src/main/java/io/logz/sender/HttpsRequestConfiguration.java

```diff
@@ -83,6 +83,7 @@ public HttpsRequestConfiguration(String logzioToken,
         private String logzioListenerUrl = "https://listener.logz.io:8071";
         private String logzioToken;
         private boolean compressRequests = false;
+        private SenderStatusReporter reporter;
```

Comment:
A configuration object (i.e. a data structure) should not contain objects to act upon like a SenderStatusReporter